packages/core/strapi/package.json

```diff
@@ -132,7 +132,8 @@
     "qs": "6.11.1",
     "resolve-cwd": "3.0.0",
     "semver": "7.5.4",
-    "statuses": "2.0.1"
+    "statuses": "2.0.1",
+    "ts-zen": "0.1.2"
```

Comment:
From what I understand this is a package of yours? Should we add it to the monorepo instead?

Reply:
Yes, this is a personal side project that I ended up working on during my free time.
The goal here is to use it as a way to test our types. The library itself is something I intend to hopefully keep as personal work on my GH :eyes:

Reply:
Really cool! ✨

I am just suggesting to not introduce that as a dependency from npm, because it is a personal project and rather move the code of `ts-zen` into the monorepo, for a couple of reasons such as you leaving Strapi one day, having visibility over the codebase, licensing questions ....

Reply:
It's MIT license like Strapi so all good there. Worst case, @Convly leaves and/or abandons the project and we add it to the monorepo as a util with accreditation.

I'll let @Convly work out the political issues around this, I know we usually don't accept low-popularity dependencies but this seems like a reasonable exception to me.

Reply:
If we don't want to move it to the monorepo, I would prefer if the repository would be created in the strapi org and published from there.

What do you think about that?